tests/testthat/test-iv-robust.R

```diff
@@ -163,7 +163,7 @@ test_that("iv_robust matches AER + ivpack", {
   dat$x1_c <- dat$x
   ivdefr <- iv_robust(y ~ x + x1_c| z + z2, data = dat, se_type = "HC0")
   ivdef <- ivreg(y ~ x + x1_c| z + z2, data = dat)
-  ivdefse <- robust.se(ivdef)
+  capture_output(ivdefse <- robust.se(ivdef))
```

Comment:
why all the capture outputs?

Reply:
shouldn't need them.

Reply:
ivpack prints whether the SEs are clustered or just robust w/ every call.

Reply:
https://github.com/cran/ivpack/blob/master/R/robust.se.R#L4

🤦‍♂️

Reply:
Yep.